### ISSUE TITLE
AttributeError: type object 'threadpool_limits' has no attribute 'wrap'

### ISSUE THREAD

**Issue by e5k** [jgrss/geowombat#226]
Hi - thanks for the great package. I am getting this error when importing geowombat:

```AttributeError: type object 'threadpool_limits' has no attribute 'wrap'```

I am running Conda on MacOS with python 3.9. Please let me know what other inf you need!

### Full error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-0d8a7e1ed56a> in <module>
----> 1 import geowombat as gw

~/.conda/envs/volcgis/lib/python3.9/site-packages/geowombat/__init__.py in <module>
      3 
      4 from . import config
----> 5 from .core.api import open
      6 from .core.api import load
      7 from .core.api import series

~/.conda/envs/volcgis/lib/python3.9/site-packages/geowombat/core/__init__.py in <module>
      1 from ..backends import transform_crs
----> 2 from .io import apply
      3 from .io import save
      4 from .io import to_raster
      5 from .io import to_netcdf

~/.conda/envs/volcgis/lib/python3.9/site-packages/geowombat/core/io.py in <module>
    351 
    352 
--> 353 @threadpool_limits.wrap(limits=1, user_api='blas')
    354 def _write_xarray(*args):
    355     """Writes a DataArray to file.

AttributeError: type object 'threadpool_limits' has no attribute 'wrap'
``

**Comment by mmann1123** [comment]
Can you run:
`conda update geowombat -c conda-forge`

then try again?

**Comment by mmann1123** [comment]
if that doesn't do the trick please send the output of:
`conda list`

**Comment by jgrss** [comment]
Hi @e5k,

I think you need to upgrade `threadpoolctl` by `pip install -U threadpoolctl`. 

I suspect that you had `threadpoolctl` installed prior to `geowombat` and our `setup.cfg` requirement for `threadpoolctl` has no version requirement. If the above upgrade works for you then we will pin `threadpoolctl` to a minimum version.

**Comment by jgrss** [comment]
@e5k thanks for raising this issue. See #227 for the fix and `v2.0.16` for the update. Please reopen if this did not solve it on your end.

**Comment by e5k** [comment]
Hey @jgrss and @mmann1123 - thanks for your quick response and feedback. Sorry I was slow to answer, but looking forward to playing with you library!
